cmd/restic/cmd_backup.go

```diff
@@ -100,6 +101,13 @@ type BackupOptions struct {
 var backupOptions BackupOptions
 
 func init() {
+	//set FileReadConcurrency to 4 is not set in env
+	filereadconcurrency, err := strconv.Atoi(os.Getenv("RESTIC_FILE_READ_CONCURRENCY"))
+	if err != nil {
+		filereadconcurrency = 2
+	} else if filereadconcurrency < 1 {
+		filereadconcurrency = 1
+	}
```

Comment:
Sseems to me that `filereadconcurrency` is not used later in your code...

Reply:
Ah, that must have gotten clobbered in a merge.  :grimacing: 

I'll fix that up in my future changes.